src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Unix.cs

```diff
@@ -13,7 +13,22 @@ internal static CultureInfo GetUserDefaultCulture()
                 return CultureInfo.InvariantCulture;
 
             CultureInfo cultureInfo;
-            if (CultureData.GetDefaultLocaleName(out string? localeName))
+            string? localeName;
+            bool result = false;
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            if (GlobalizationMode.Hybrid)
```

Comment:
Should we bump the hybrid logic to `GetDefaultLocaleName` in `CultureData.Icu.cs` to be more consistent with calling the `Interop.Globalization.HybridImplementation` `Interop.Globalization.NonHybridImplementation` in the same area.